src/lwc/geBatchWizard/geBatchWizard.js

```diff
@@ -209,8 +220,22 @@ export default class geBatchWizard extends NavigationMixin(LightningElement) {
         });
     }
 
-    @wire(getRecordCreateDefaults, { objectApiName: '$dataImportBatchName' })
-    dataImportBatchCreateDefaults;
+    appendTrueFalsePicklistToElement() {
+        this.formSections.forEach(section => {
+            if (section.elements) {
+                section.elements.forEach(element => {
+                    const fieldMapping = GeFormService.getFieldMappingWrapper(element.dataImportFieldMappingDevNames[0]);
+
+                    if(isNotEmpty(fieldMapping)) {
+                        if(isTrueFalsePicklist(fieldMapping)) {
+                            element.picklistOptionsOverride = trueFalsePicklistOptions();
+                        }
+                        element.isCheckboxToCheckbox = isCheckboxToCheckbox(fieldMapping);
```

Comment:
In the batch wizard, we don't render checkboxes at all since the only operation you can do is set a default value, as opposed to template builder where you can rearrange field order etc. We need to identify those fields in some way, and normal fields (text/lookup) wouldn't have the picklistOptionsOverride populated. This one might be worth talking through a bit!

Reply:
We talked through this and ended up making the decision to allow GeFormService to retrieve fieldMappings w/o retrieving a template.